src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/VariantsToTable.java

```diff
@@ -122,6 +123,7 @@
 public final class VariantsToTable extends VariantWalker {
     public final static String SPLIT_MULTI_ALLELIC_LONG_NAME = "split-multi-allelic";
     public final static String SPLIT_MULTI_ALLELIC_SHORT_NAME = "SMA";
+    public static final String NUMERIC_GT_FULLNAME = "numeric-gt";
```

Comment:
perhaps "use-numeric-gt" to make it verbed appropriately?

Reply:
done